torch/overrides.py

```diff
@@ -196,6 +196,7 @@ def get_ignored_functions() -> Set[Callable]:
         torch.nn.functional.sigmoid,
         torch.nn.functional.hardsigmoid,
         torch.nn.functional.tanh,
+        torch.profiler.profiler._profile_using_dynolog,
```

Comment:
nit: kind of a weird place in the list to put this.

Reply:
placed it in alphabetical order.